src/components/Migration/__tests__/MigrationModal.unit.test.jsx

```diff
@@ -9,13 +9,8 @@ import MigrationModal from "../MigrationModal";
 
 describe("<MigrationModal/>", () => {
   it("should render closed component", () => {
-    const { container } = render(<MigrationModal open={false} handleClose={() => console.log("handleClose")} />);
-    expect(container).toMatchSnapshot();
-  });
```

Comment:
snapshots like this seem small enough to still be useful?

Reply:
This snapshot actually wasn't doing anything though. When i looked at the snapshot output it was just an empty div.. so it wasn't actually testing anything. 
Moving it to check for text on the screen seems like a safer way to test, since we're checking for an expected return value. 
Previous snapshot was expecting a match of a snapshot, which was empty.

Reply:
OHHHHH lol. OK, I got you now